corehq/apps/users/migrations/0041_migrate_reprs.py

```diff
@@ -0,0 +1,14 @@
+from django.db import migrations
+
+from corehq.util.django_migrations import run_once_off_migration
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0040_add_repr_and_changed_by_repr'),
+    ]
+
+    operations = [
+        run_once_off_migration('migrate_reprs')
```

Comment:
This is also has skip_on_fresh_install which is great.

Just so I understand this correctly, what happens if the command fails?
The user is prompted with a message that they need to run the management command by themselves and till then they can't proceed? And once they run the command, the migration would be run again?